src/App.js

```diff
@@ -0,0 +1,36 @@
+const { once } = require('events');
+const path = require('path');
+const express = require('express');
+const config = require('./config.js').config();
```

Comment:
It's not that bad, but in this line, you try to mix an import with a constant declaration. It won't be possible when using ES6 imports, so I would really like to separate importing the `config` module and reading `DEFAULT_CONFIG` from it.